src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const liArray = [...document.querySelectorAll('li')];
+
+for (let i = liArray.length - 1; i >= 0; i--) {
+  if (liArray[i].querySelector('ul')) {
+    const heading = liArray[i].firstChild.textContent.trim();
```

Comment:
This code can be shortened:
1. create new span element;
2. prepend it to 'li' element;
3. Use something like 'span.append(span.nextSibling)' to move the text (which is behind the span) inside the span.

This also will let you get rid of this check ''' if (liArray[i].querySelector('ul')) {...."

Reply:
Thanks! Good advice 👍 
But I'll leave my check, because it allows me to cover with <span> only list items, which are ul-headers (not every single span in the list)